app/src/main/java/com/omen273/crossLingo/GameActivity.kt

```diff
@@ -207,6 +213,26 @@ class GameActivity : AppCompatActivity(), CrosswordView.OnLongPressListener,
         if (crosswordView.state?.isCompleted ?: return) showFinishGameDialog(true)
         keyboard_ga.inputConnection = crosswordView.onCreateInputConnection(EditorInfo())
         freeClueRestart()
+
+        if (ttsEnabled)
+        {
+            initTTS()
```

Comment:
Please check that it is possible to use lateinit in this way. You initialize it conditionally/

Reply:
I think it is better to use nullable TTS. Because you create it conditionally so you can eleiminate ttsEnabled

Reply:
done